industry/products.md

```diff
@@ -17,11 +17,13 @@ They vary by:
 and other features, as well as underlying architecture.
 
 
-An advantage of using a machine translation provider is that the service is already working and can be integrated immediately. An alternative to using a machine translation provider is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
+Some of the most popular MT providers offer off-the-shelf solutions, i.e. trained MT models that are ready-to-use and can be integrated immediately. An alternative is to train a model from scratch, deploy it and maintain it.
```

Comment:
Agreed. The reason why I tried to rephrase this and add some explanation is that I was afraid that the reader might be lost when reading about "MT providers", "raw models", "production systems", "models". Especially "machine translation providers" seems a bit ambiguous to me, since it's often used to refer to the company providing MT solutions, and the company might offer different solutions, not only the ready-to-use ones. My suggestion would be to find a unique term for the ready-to-use solution and one for the model to be trained from scratch so that these can be used consistently. These are just suggestions, of course. If it's just me, we can leave the original!